qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -10,10 +12,36 @@
             image_num = 31
             image_parent_bus = pci_bridge1
             sub_test_type = dd_test
-            dd_bs = 1K
-            dd_count = 1000
             variants:
                 - write:
                     dd_if = ZERO
                 - read:
                     dd_of = NULL
+        - hotplug_unplug:
+            only Linux
+            type = pci_bridge_hotplug_unplug
+            image_num = 3
+            images += " stg0 stg1 stg2"
+            image_name_stg0 = images/storage0
+            image_name_stg1 = images/storage1
+            image_name_stg2 = images/storage2
+            image_size_stg0 = 100M
+            image_size_stg1 = 100M
+            image_size_stg2 = 100M
+            boot_drive_stg0 = no
+            boot_drive_stg1 = no
+            boot_drive_stg2 = no
+            force_create_image_stg0 = yes
+            force_create_image_stg1 = yes
+            force_create_image_stg2 = yes
+            remove_image_stg0 = yes
+            remove_image_stg1 = yes
+            remove_image_stg2 = yes
+            blk_extra_params_stg0 = "serial=TARGET_DISK0"
+            blk_extra_params_stg1 = "serial=TARGET_DISK1"
+            blk_extra_params_stg2 = "serial=TARGET_DISK2"
```

Comment:
Sorry. I don't think it is meaningful to move these image params generating part to .py file, because this part is not a bulk of codes that bringing blurring (only 18 lines and fit in a single screen which is normally more than 40 lines) and current style is more clear. Thanks!